src/components/app/details/triggerView/workflow/nodes/triggerCINode.tsx

```diff
@@ -129,10 +138,11 @@ export class TriggerCINode extends Component<TriggerCINodeProps> {
                 <div className="workflow-node__title flex">
                     {/* <img src={build} className="icon-dim-24 mr-16" /> */}
                     <div className="workflow-node__full-width-minus-Icon">
-                        <span className="workflow-node__text-light">{this.props.isJobView ? 'Job' : 'Build'}</span>
+                        {!this.props.isJobView && <span className="workflow-node__text-light">Build</span>}
```

Comment:
Don't we need the text `job` here?

Reply:
In design it was not there , that's why removed it.